src/pages/stacks-blockchain/overview.md

```diff
@@ -50,28 +50,28 @@ Smart contracts on the Stacks 2.0 blockchain are written in a new language calle
 
 ### Stacks Mining
 
-Mining is required to make the network trustworthy and secure, by verifying transaction information. Mining is the act of participation in the consensus mechanism of the Stacks 2.0 blockchain, called Proof of Transfer (PoX). To incentivize miners, who have to run mining software on their machine, they receive freshly minted Stacks (STX) tokens.
+Mining is required to make the network usable, trustworthy, and secure. Miners verify incoming transactions and participate in the consensus mechanism of the Stacks 2.0 blockchain, called Proof of Transfer (PoX). To incentivize mining, miners receive freshly minted Stacks (STX) tokens.
 
 [@page-reference | inline]
 | /mining
 
 ### Proof of Transfer (PoX)
 
-Proof of Transfer (PoX) is the consensus mechanism that leverages the security and stability of Bitcoin to create new blockchains, without modifying the Bitcoin protocol.
+Proof of Transfer (PoX) is a novel consensus mechanism that leverages the security and stability of Bitcoin to create a new blockchain, without modifying the Bitcoin protocol. 
 
-PoX makes it possible to reward holders of Stacks Token (STX) with Bitcoin: instead of spending energy, Stacks miners transfer Bitcoin to holders of Stacks Token (STX) to mine blocks. An implementation of this is called **Stacking**.
+STX miners transfer Bitcoin to other STX holders in order to mine blocks. This BTC reward mechanism is called **Stacking**.
 
 ### Stacking
 
-Stackers are Stacks (STX) Token holders who help secure the network by effectively acting as validators on the network, incentivizing honest behavior on the part of miners. Stackers have to lock up Stacks (STX) for a certain period of time. As a reward, Stackers receive the Bitcoin that miners transfer as part of Proof of Transfer (PoX).
+STX token holders can help secure the network by participating in Stacking and locking up their STX tokens for a certain period of time. As a reward for this commitment, Stackers receive Bitcoin that miners transfer as part of Proof of Transfer (PoX).
 
 -> Proof of Transfer and Stacking are in active development and are coming soon
 
 ## Protocol specification
 
-The Stacks 2.0 blockchain specification consists a set of proposals, called Stacks improvement proposals (SIPs). Each SIP is aimed at describing the implementation of a part of the Stacks 2.0 blockchain. Together, all proposals contain concise technical specifications of features or standards and the rationale
-behind it.
+The Stacks 2.0 blockchain specification consists a set of proposals, called Stacks improvement proposals (SIPs). Each SIP is describes the implementation of a part of the Stacks 2.0 blockchain. Together, all proposals contain concise technical specifications of features or standards and the rationale
```

Comment:
`Each SIP is describes the implementation` > `Each SIP describes the implementation`